src/beamlime/services/dashboard.py

```diff
@@ -157,6 +157,23 @@ def _setup_layout(self) -> None:
                 value=[45, 55],
                 marks={i: str(i) for i in range(0, 101, 20)},
             ),
+            html.Label('Time-of-arrival range (us)'),
+            dcc.Checklist(
+                id='toa-checkbox',
+                options=[
+                    {'label': 'Filter by time-of-arrival [μs]', 'value': 'enabled'}
```

Comment:
It seems it's microseconds here and nanoseconds above?

Reply:
Where?

Reply:
The text in the `html.Label` is `'Time-of-arrival range (us)'`, and the unit label in the square brackets is `Filter by time-of-arrival [μs]`?

Unless this is not the same time as in the `_convert_toa_range` function?

Reply:
Not sure I understand, `us` is just another spelling of microsecond?

Reply:
What I meant was that on L149-150 in `detector_data_handler.py` you have converted them to nanoseconds?
By 'above' I meant further up the page, not in the same file, sorry.

Reply:
What is wrong with converting microseconds to nanoseconds? The dashboard uses microseconds, since nanoseconds are a bit unwieldy for a user to think about.